model/work.py

```diff
@@ -141,13 +141,21 @@ class Work(Base):
     presentation_edition_id = Column(Integer, ForeignKey('editions.id'), index=True)
 
     # One Work may have many associated WorkCoverageRecords.
-    coverage_records = relationship("WorkCoverageRecord", backref="work")
+    coverage_records = relationship(
+        "WorkCoverageRecord", backref="work",
+        cascade="all, delete-orphan"
+    )
```

Comment:
Without this, we can end up with dangling WorkCoverageRecords that have no associated Work.